### PR TITLE
Adjust smartcard_configure_cert_checking OVAL for other distros

### PR DESCRIPTION
**Pull request by dodys**
#### Description:

- smartcard_configure_cert_checking OVAL was SUSE specific, changed it so other distros can use it as well.

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/oval/shared.xml [hunk 1]
```diff
@@ -1,34 +1,26 @@
 <def-group>
   <definition class="compliance" id="smartcard_configure_cert_checking" version="4">
-    <metadata>
-      <title>Enable Smart Card Login</title>
-      <affected family="unix">
-        <platform>multi_platform_sle</platform>
```
Comment (jan-cerny) [inline linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/oval/shared.xml:6]: I think that if you enable this on all platforms you should remove the rhel8 from the prodtype in the rule becasue on rhel8 neither the configuration file nor the package exist.

Comment (dodys) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/oval/shared.xml:6]: So, rhel8 is in the prodtype but the rule is not in any rhel8 profile. That's probably a leftover?
If that's right, I can remove rhel8 from the prodtype in a new commit.

Comment (dodys) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/oval/shared.xml:6]: I've removes rhel8 from prodtype in a new commit.

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**